wizard/WizardNav.qml

```diff
@@ -89,19 +88,25 @@ GridLayout {
         Layout.fillWidth: true
         color: "transparent"
 
-        RowLayout {
+        PageIndicator {
             id: wizardProgress
-            spacing: 0
-            width: 100  // default, dynamically set later
-            height: 30
+            currentIndex: menuNav.progress
+            count: menuNav.progressSteps
             anchors.horizontalCenter: parent.horizontalCenter
             anchors.verticalCenter: parent.verticalCenter
+            spacing: 25
+            delegate: Rectangle {
+                    implicitWidth: 10
+                    implicitHeight: 10
+                    radius: 10
+                    color: index === menuNav.progress ? MoneroComponents.Style.defaultFontColor : MoneroComponents.Style.progressBarBackgroundColor
```

Comment:
To get the same behaviour as previously:

```suggestion
                    color: index <= menuNav.progress ? MoneroComponents.Style.defaultFontColor : MoneroComponents.Style.progressBarBackgroundColor
```

Reply:
I tried your suggestion, but it doesn't work in the first pages (WizardCreateWallet1.qml and WizardRestoreWallet1.qml). It seems that the PageIndicator doesn't recognize 1 as index in the first page (instead of index 0), so that if you set progress: 1 in these pages it will always display as if it were the 2nd page (two white dots).

Reply:
Seems to work on my system.

<img width="1092" alt="Screenshot 2020-04-22 at 22 54 48" src="https://user-images.githubusercontent.com/7697454/80032977-46036f80-84ec-11ea-9fbf-fffb56b6c98d.png">

Reply:
In Linux (Ubuntu):

WizardNav.qml
```
        PageIndicator {
            id: wizardProgress
            currentIndex: menuNav.progress
            count: menuNav.progressSteps
            anchors.horizontalCenter: parent.horizontalCenter
            anchors.verticalCenter: parent.verticalCenter
            spacing: 25
            delegate: Rectangle {
                    implicitWidth: 10
                    implicitHeight: 10
                    radius: 10
                    color: index <= menuNav.progress ? MoneroComponents.Style.defaultFontColor : MoneroComponents.Style.progressBarBackgroundColor
            }
        }
```

WizardRestoreWallet1.qml:
```
            WizardNav {
                id: nav
                progressSteps: appWindow.walletMode <= 1 ? 3 : 4
                progress: 1
```
Result:
![image](https://user-images.githubusercontent.com/45968869/80033822-a646e100-84ed-11ea-8219-f3658c2dd21a.png)

Reply:
Looks like a Qt bug. You can set it to `===` and add a comment:

```
@TODO: Qt 5.10+ replace === with <=
```